internal/services/engines/nodejs/regular/regular.go

```diff
@@ -364,3 +364,35 @@ func NewNodeJSRegularUsingCommandLineArguments() text.TextRule {
 		},
 	}
 }
+
+func NewNodeJSRegularRedirectToUnknownPath() text.TextRule {
+	return text.TextRule{
+		Metadata: engine.Metadata{
+			ID:          "71c4c404-c796-4ce7-98f6-936b24248776",
+			Name:        "Redirect to unknown path",
+			Description: "Sanitizing untrusted URLs is an important technique for preventing attacks such as request forgeries and malicious redirections. Often, this is done by checking that the host of a URL is in a set of allowed hosts. For more information checkout the CWE-20 (https://cwe.mitre.org/data/definitions/20.html) advisory.",
+			Severity:    severities.High.ToString(),
+			Confidence:  confidence.Low.ToString(),
+		},
+		Type: text.Regular,
+		Expressions: []*regexp.Regexp{
+			regexp.MustCompile(`redirect\((?:\w)`),
+		},
+	}
+}
```

Comment:
Can we make it pluggable?
I mean, the structure is the same in all the implementations:
ID, Name, Description, Severity, Confidence, and the regex.
Maybe it is an opportunity to check it in the future improvements.

Reply:
I don't understand what do you mean with "can we make it pluggable".  This pr introduce a new standard rule, [users can add custom rules on their own](https://horusec.io/docs/tutorials/how-to-adding-custom-rules/)

Reply:
Put the rules on code we can compile them into the final binary. If we use a configuration file to write standard rules we need to make this config available too, which will make the installation more complicated.

Reply:
the rules are created by horusec are inside the tool, to avoid possible external connection problems and ensure agility in reading the rule

Ex.: on my CI/CD it is not possible to acquire data from outside the internal network.

Reply:
@wiliansilvazup @matheusalcantarazup 
Please, take a look at:
https://pkg.go.dev/embed

Reply:
Whats the difference between write the rules on Go code and embed the file with rules into the final binary? The two options will be statically into the binary.

Reply:
I'll be marking the conversation as resolved so we can merge the PR.